pkg/services/accesscontrol/resourcepermissions/service.go

```diff
@@ -108,12 +108,13 @@ type Service struct {
 	api     *api
 	license licensing.Licensing
 
-	options     Options
-	permissions []string
-	actions     []string
-	sqlStore    db.DB
-	teamService team.Service
-	userService user.Service
+	options            Options
+	permissions        []string
+	actions            []string
+	fineGrainedActions []string // defines a set of actions that are allowed to be directly used in a custom set of actions
```

Comment:
pretty sure we don't need this one. We can use already existing actions for validation. That should be a flat list of all valid actions

Reply:
The main issue is that we require the set of allowed actions to be defined for Grafana enterprise. This particularly was a strategy to avoid using the set of all possible actions, but I would definitely be interested in other strategies to accomplish that.